web-server/v0.4/src/utils/parse.js

```diff
@@ -77,9 +78,17 @@ export const parseIterationData = results => {
           }
 
           iterationData.forEach(hostMetadata => {
-            const columnHost = `client_hostname:${hostMetadata.client_hostname}-server_hostname:${
-              hostMetadata.server_hostname
-            }-server_port:${hostMetadata.server_port}`;
+            const clientHostname = hostMetadata.client_hostname
+              ? `client_hostname:${hostMetadata.client_hostname}`
+              : ``;
+            const serverHostname = hostMetadata.server_hostname
+              ? `server_hostname:${hostMetadata.server_hostname}`
+              : ``;
+            const serverPort = hostMetadata.server_port
+              ? `server_port:${hostMetadata.server_port}`
+              : ``;
```

Comment:
If all of the above resolve to empty strings, how should this function behave?

Reply:
An empty column cell would be rendered in this case. We make an assumption that each iteration has a hostname or port assigned to it but in the case there isn't, we make use of the `columnPrefix` as the dataIndex used for that column render.